frontend/src/pages/notebookController/screens/server/StopServerModal.tsx

```diff
@@ -78,16 +104,33 @@ const StopServerModal: React.FC<StopServerModalProps> = ({ notebooksToStop, onNo
 
   return (
     <Modal
-      aria-label="Stop server modal"
+      aria-label="Stop workbench modal"
       appendTo={document.body}
       variant={ModalVariant.small}
-      title={`Stop ${textToShow}`}
+      title={`Stop ${textToShow}?`}
       isOpen
       showClose
       onClose={onClose}
       actions={modalActions}
     >
-      Are you sure you want to stop {textToShow}? Any changes made without saving will be lost.
+      <Stack hasGutter>
```

Comment:
cc @srtanish1992 This will likely have conflicts with one of your PRs, just FYI.